project/Build.scala

```diff
@@ -1132,8 +1138,7 @@ object Build {
     def asDottyRoot(implicit mode: Mode): Project = project.withCommonSettings.
       aggregate(`dotty-interfaces`, dottyLibrary, dottyCompiler, dottyDoc, dottySbtBridgeReference).
       bootstrappedAggregate(`scala-library`, `scala-compiler`, `scala-reflect`, scalap, `dotty-language-server`).
-      dependsOn(dottyCompiler).
-      dependsOn(dottyLibrary).
+      dependsOn(dottyCompiler, dottyLibrary, dottyBench).
```

Comment:
Please don't do this. sbt currently downloads the dotty artifact and it's dependencies, do everyone will download dotty-bench if you add it as a dependency

Reply:
Can we add something like a `noPublish` setting to `dottyBench` so that is is not included in the dotty artefact?

Reply:
If you don't publish it, it will still be listed as a dependency, and downloading dotty will fail when the dependency is not found.

Reply:
Same if we put it in `aggregate` and not `dependsOn`?

Reply:
I will remove this part